src/plugins/intel_cpu/src/cpu_types.cpp

```diff
@@ -13,207 +13,207 @@ using TypeToNameMap = InferenceEngine::details::caseless_unordered_map<std::stri
 
 static const TypeToNameMap& get_type_to_name_tbl() {
     static const TypeToNameMap type_to_name_tbl = {
-            { "Constant", Type::Input },
```

Comment:
Is this caused by the CI check? if no, please revert.

Reply:
It was caused by the clang fix on this file, reverted back